styles.css

```diff
@@ -0,0 +1,1234 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&display=swap');
+
+* {
+  font-family: poppins, Arial, Helvetica, sans-serif;
+  font-size: 32px;
+  box-sizing: border-box;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  align-content: center;
+  height: 8vh;
+  width: 70vw;
+  margin-left: 0.2rem;
+  padding-top: 8px;
+}
+
+.header .menu-icon {
+  position: relative;
+  top: 10%;
+  margin-right: 30%;
+  width: 0.7rem;
+  height: 0.7rem;
+  left: 5%;
+}
+
+.header .website-logo {
+  width: 2.4rem;
+  height: 1rem;
+  position: relative;
+  top: -0.2rem;
+}
+
+.header .desktop-menu li {
+  display: none;
+}
+
+.header .mobile-menu {
+  position: absolute;
+  list-style-type: none;
+  display: none;
+  width: 100vw;
+  height: 100%;
+  background: lightgrey;
+  z-index: 10;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+}
```

Comment:
Great design @marvin-nyalik 🤝

- [ ] Kindly ensure that while on the mobile view of the About page, when the hamburger menu is clicked, the mobile menu should contain a `home` button which whenever clicked will navigate us to the home page.
![Your about button](https://github.com/marvin-nyalik/SoccerFestival/assets/63917725/1906d0e6-0753-4a74-87f9-fbea61f5d2a7)